sql/hive/src/test/scala/org/apache/spark/sql/hive/test/TestHive.scala

```diff
@@ -647,3 +647,29 @@ private[sql] class TestHiveSessionStateBuilder(
 
   override protected def newBuilder: NewBuilder = new TestHiveSessionStateBuilder(_, _)
 }
+
+private[hive] object HiveTestJars {
+  private val repository = "https://repository.apache.org/content/repositories/releases/"
+
+  def getHiveContribJar: File =
+    getJarFromUrl(s"${repository}org/apache/hive/hive-contrib/" +
+      s"${HiveUtils.builtinHiveVersion}/hive-contrib-${HiveUtils.builtinHiveVersion}.jar")
+  def getHiveHcatalogCoreJar: File =
+    getJarFromUrl(s"${repository}org/apache/hive/hcatalog/hive-hcatalog-core/" +
+      s"${HiveUtils.builtinHiveVersion}/hive-hcatalog-core-${HiveUtils.builtinHiveVersion}.jar")
+
+  private def getJarFromUrl(urlString: String): File = {
+    val fileName = urlString.split("/").last
+    val hiveTestJarsDir = new File("/tmp/test-spark/hiveTestJars")
```

Comment:
I suppose you could use `withTempDir` or something to make sure the JARs are in a standard temp place, but probably does not matter.

Reply:
Changed it to `Utils.createTempDir()`: Create a temporary directory inside the given parent directory. The directory will be automatically deleted when the VM shuts down.